external/Kconfig

```diff
@@ -16,4 +16,5 @@ source "$EXTERNALDIR/protobuf/Kconfig"
 source "$EXTERNALDIR/wakaama/Kconfig"
 source "$EXTERNALDIR/WICED/Kconfig"
 source "$EXTERNALDIR/libopus/Kconfig"
-source "$EXTERNALDIR/stress_tool/Kconfig"
\ No newline at end of file
+source "$EXTERNALDIR/stress_tool/Kconfig"
+source "$EXTERNALDIR/error_report/Kconfig"
```

Comment:
Could you add it by Arphabet order?
The rest of the list is listed by Arphabet order.

Reply:
will carry out the change.